apps/test/unit/code-studio/components/progress/UnitCalendarLessonChunkTest.jsx

```diff
@@ -0,0 +1,209 @@
+import {expect, assert} from '../../../../util/reconfiguredChai';
+import React from 'react';
+import {shallow} from 'enzyme';
+import UnitCalendarLessonChunk from '@cdo/apps/code-studio/components/progress/UnitCalendarLessonChunk';
+import color from '@cdo/apps/util/color';
+
+const sampleLessonChunk = {
+  id: 1,
+  title: 'test',
+  duration: 10,
+  assessment: true,
+  unplugged: true,
+  isStart: true,
+  isEnd: true,
+  isMajority: true,
+  url: 'https://www.google.com/'
+};
+
+describe('UnitCalendarLessonChunk', () => {
+  it('is purple border with grey text when is assessment', () => {
+    const wrapper = shallow(
+      <UnitCalendarLessonChunk
+        minuteWidth={1}
+        lessonChunk={{
+          ...sampleLessonChunk,
+          assessment: true
+        }}
+        isHover={false}
+        handleHover={() => console.log('hover')}
+      />
+    );
+
+    assert.propertyVal(
+      wrapper.find('a').prop('style'),
+      'border',
+      '2px solid ' + color.purple
+    );
+    assert.propertyVal(wrapper.find('a').prop('style'), 'color', '#333');
+  });
+  it('is purple background with white text when is assessment and being hovered', () => {
+    const wrapper = shallow(
+      <UnitCalendarLessonChunk
+        minuteWidth={1}
+        lessonChunk={{
+          ...sampleLessonChunk,
+          assessment: true
+        }}
+        isHover={true}
+        handleHover={() => console.log('hover')}
+      />
+    );
+
+    assert.propertyVal(
+      wrapper.find('a').prop('style'),
+      'border',
+      '2px solid ' + color.purple
+    );
+    assert.propertyVal(
+      wrapper.find('a').prop('style'),
+      'backgroundColor',
+      color.purple
+    );
+    assert.propertyVal(wrapper.find('a').prop('style'), 'color', 'white');
+  });
+  it('is teal border with grey text when is assessment', () => {
+    const wrapper = shallow(
+      <UnitCalendarLessonChunk
+        minuteWidth={1}
+        lessonChunk={{
+          ...sampleLessonChunk,
+          assessment: false
+        }}
+        isHover={false}
+        handleHover={() => console.log('hover')}
+      />
+    );
+
+    assert.propertyVal(
+      wrapper.find('a').prop('style'),
+      'border',
+      '2px solid ' + color.teal
+    );
+    assert.propertyVal(wrapper.find('a').prop('style'), 'color', '#333');
+  });
```

Comment:
nit: add an extra line between tests (really helps to see where one test ends and the next begins!)